src/main/resources/locale/locale_de.properties

```diff
@@ -106,76 +106,76 @@ Combat.BeastLoreHorseJumpStrength = &3Pferde-Sprungst\u00E4rke (&aMaximal {0} Bl
+Commands.Description.mcchatspy = Schalte den Party Chat-Spy an oder aus.
+Commands.Description.mcconvert = Konvertiert Datenbanktypen oder Erfahrungsformeln.
+Commands.Description.mccooldown = Sieh alle F\u00E4higkeiten-Cooldowns.
+Commands.Description.mcgod = Schalte mcMMO godmode an oder aus.
+Commands.Description.mchud = \u00C4ndere deinen HUD Stil.
+Commands.Description.mcmmo = Zeige eine kurze Beschreibung \u00FCber mcMMO.
+Commands.Description.mcnotify = Schalte F\u00E4higkeiten-Hinweise im Chat an oder aus.
 Commands.Description.mcpurge = Bereinige die mcMMO Datenbank von Spielern die {0} Monate nicht online waren oder keine Level haben.
-Commands.Description.mcrank = Zeige das Skill Ranking f\u00FCr einen Spieler
-Commands.Description.mcrefresh = Aktualisiere alle Cooldowns
-Commands.Description.mcremove = Entferne einen Benutzer aus der Datenbank
-Commands.Description.mcscoreboard = Verwalte deine Skill \u00DCbersicht
-Commands.Description.mcstats = Zeige deine Skill Level und Erfahrung
-Commands.Description.mctop = Zeige die Skill Bestenlisten
-Commands.Description.mmocompat = Information about mcMMO and whether or not its in compatibility mode or fully functional.
-Commands.Description.mmodebug = (De)aktiviere den Debugging-Modus, welcher n\u00FCtzliche Informationen ausgibt, wenn du einen Block schl\u00E4gst
-Commands.Description.mmoedit = Editiere die Skill Level eines Spielers
-Commands.Description.mmoinfo = Lese Details \u00FCber einen Skill oder andere Funktion
-Commands.Description.mmoshowdb = Zeige den Namen der aktuellen Datenbank (zur Benutzung mit /mmoupdate)
-Commands.Description.mmoupdate = Kopiere Daten von einer alten Datenbank zur aktuell benutzen
-Commands.Description.mmoxpbar = Player settings for mcMMO XP bars
-Commands.Description.party = \u00C4ndere verschiedene Party Einstellungen
-Commands.Description.partychat = Schalte den mcMMO Party Chat an oder aus oder sende Party Nachrichten
-Commands.Description.ptp = Teleportiere zu einem Party Mitglied
-Commands.Description.skillreset = Setze die Skills eines Spielers zur\u00FCck
-Commands.Description.vampirism = Schalte Vampirismus an oder aus bzw. \u00E4ndere den Vampirismus Prozentsatz
-Commands.Description.xplock = Setze deine Erfahrungs-Leiste auf einen bestimmten Skill fest
-Commands.Description.xprate = \u00C4ndere die Erfahrungs Rate oder starte ein Erfahrungs Event
+Commands.Description.mcrank = Zeige das Skill-Ranking f\u00FCr einen Spieler.
+Commands.Description.mcrefresh = Aktualisiere alle Cooldowns.
+Commands.Description.mcremove = Entferne einen Benutzer aus der Datenbank.
+Commands.Description.mcscoreboard = Verwalte deine Skill-\u00DCbersicht.
+Commands.Description.mcstats = Zeige deine Skill-Level und Erfahrung.
+Commands.Description.mctop = Zeige die Skill-Bestenlisten.
+Commands.Description.mmocompat = Informationen darüber, ob mcMMO im Kompatibilitätsmodus oder voll funktionsfähig ist.
```

Comment:
ä is not properly encoded in this string.

Reply:
Thank you, I actually missed some of the UTF-8-Encoding. I will fix that today!

Reply:
I replaced all the missing Umlaute with their UTF-8 encoding (forgot about that, sorry). I never worked with Github before to be honest, but I think you should be able to see the new commit. :)

Reply:
Yep, whenever you commit to the branch you opened the pull request with, your changes/commits will automatically update in the pull request too!
It's looking very good to me now - as a fellow native speaker.